daprdocs/content/en/operations/hosting/kubernetes/kubernetes-production.md

```diff
@@ -6,11 +6,13 @@ weight: 40000
 description: "Recommendations and practices for deploying Dapr to a Kubernetes cluster in a production-ready configuration"
 ---
 
-## Cluster capacity requirements
+## Cluster and capacity requirements
+
+Dapr is supported on Kubernetes 1.23 or higher.
```

Comment:
@msfussell @ItalyPaleAle  Do we want to specify the version of k8s here? This will quickly become out of date.

Instead can we say we support the last X k8s versions?

Reply:
I am ok with that. In this specific case, 1.22 was an actual minimum because we rely on features that were not available before 1.21 (and not stable before 1.22).

Reply:
@ItalyPaleAle @greenie-msft does this still need to be updated?

Reply:
You folks tell me what you prefer :)

Reply:
Reference this article for K8s versions https://v1-10.docs.dapr.io/operations/support/support-release-policy/#supported-versions-of-dependencies

Reply:
I like @hhunter-ms' suggestion. Accepted :)